src/GraphQL/Types/ArgumentInformation.cs

```diff
@@ -57,12 +63,12 @@ public ArgumentInformation(ParameterInfo parameterInfo, Type sourceType, FieldTy
         /// The expected type of <see cref="IResolveFieldContext.Source"/>.
         /// Should equal <c>TSourceType</c> within <see cref="AutoRegisteringObjectGraphType{TSourceType}"/>.
         /// </summary>
-        public Type SourceType { get; }
+        public Type? SourceType { get; }
 
         /// <summary>
         /// The <see cref="Types.FieldType"/> that the query argument will be added to.
         /// </summary>
-        public FieldType FieldType { get; }
+        public FieldType? FieldType { get; }
```

Comment:
The schema builder does not know what type the source is, as far as I can tell.  It also has not built the field at the time that it attempts to build the field resolver.  The latter we could likely change.  For now, to make as little changes in the schema builder as possible, these properties (which were really just informative and not absolutely necessary before) are now nullable.